src/obsidian/augmentations/MarkdownView.d.ts

```diff
@@ -111,6 +111,7 @@ declare module "obsidian" {
         getSyncViewState(): any;
         /**
          * Get the current view type
+         * @returns "markdown"
          */
         getViewType(): string;
```

Comment:
Maybe it would be preferable to explicitly state the return type of the view, instead of via the TSDoc.
I.e.: `getViewType(): "markdown"`